services/module.mk

```diff
@@ -1,6 +1,6 @@
 module := services
 
-submodules := merak-compute scenario-manager
+submodules := proto merak-compute scenario-manager merak-agent
```

Comment:
@phudtran Please make this file to include all other components in the service folder and add MIT license.